src/test/resources/expected-classes/request/graphql-query/updateIssueMutation.txt

```diff
@@ -3,7 +3,7 @@ mutation {
     floatVal: 1.23,
     booleanVal: false,
     intVal: 42,
-    stringVal: "my-default",
+    stringVal: \"my-default\",
```

Comment:
Same here, the query itself expects these quotes, so I believe they should not be escaped.

Reply:
@joffrey-bion Fixed